linux/deploy_vm/rebuild_unattend_install_iso.yml

```diff
@@ -30,30 +30,68 @@
 
 - name: "Rebuild ISO for Ubuntu desktop"
   block:
-    - name: Extract specific files inside ISO
+    - name: "Extract specific files inside ISO"
       community.general.iso_extract:
         image: "{{ src_iso_file_path }}"
         dest: "{{ src_iso_file_dir }}"
         files:
           - "boot/grub/grub.cfg"
 
-    - name: Modify boot entry
+    - name: "Modify boot entry"
       ansible.builtin.replace:
         path: "{{ src_iso_file_dir }}/grub.cfg"
         regexp: "set timeout=[1-9][0-9]{0,1}"
         replace: "default=0\nset timeout=2"
 
-    - name: Modify boot options
+    - name: "Modify boot options"
       ansible.builtin.replace:
         path: "{{ src_iso_file_dir }}/grub.cfg"
         regexp: "file=/cdrom/preseed/ubuntu.seed maybe-ubiquity quiet splash ---"
         replace: "file=/cdrom/preseed/ubuntu.seed boot=casper debug-ubiquity automatic-ubiquity quiet splash noprompt --- console=ttyS0,115200n8"
 
-    - name: "Print the content of modified file"
+    - name: "Print the content of modified file grub.cfg"
       debug:
```

Comment:
Please use full Ansible module name `ansible.builtin.debug`

Reply:
Change it. Thanks